core/src/main/java/fr/sncf/osrd/cli/WorkerCommand.kt

```diff
@@ -7,12 +7,12 @@ import fr.sncf.osrd.api.ElectricalProfileSetManager
 import fr.sncf.osrd.api.InfraLoadEndpoint
 import fr.sncf.osrd.api.InfraManager
 import fr.sncf.osrd.api.VersionEndpoint
-import fr.sncf.osrd.api.api_v2.conflicts.ConflictDetectionEndpointV2
-import fr.sncf.osrd.api.api_v2.path_properties.PathPropEndpoint
-import fr.sncf.osrd.api.api_v2.pathfinding.PathfindingBlocksEndpointV2
-import fr.sncf.osrd.api.api_v2.project_signals.SignalProjectionEndpointV2
-import fr.sncf.osrd.api.api_v2.standalone_sim.SimulationEndpoint
-import fr.sncf.osrd.api.api_v2.stdcm.STDCMEndpointV2
+import fr.sncf.osrd.api.conflicts.ConflictDetectionEndpoint
+import fr.sncf.osrd.api.path_properties.PathPropEndpoint
+import fr.sncf.osrd.api.pathfinding.PathfindingBlocksEndpointV2
+import fr.sncf.osrd.api.project_signals.SignalProjectionEndpoint
+import fr.sncf.osrd.api.standalone_sim.SimulationEndpoint
+import fr.sncf.osrd.api.stdcm.STDCMEndpointV2
```

Comment:
Also remove V2 in namings:
There might be other occurrences, always good to get, and OK to be handled with later PRs.
```suggestion
import fr.sncf.osrd.api.pathfinding.PathfindingBlocksEndpoint
import fr.sncf.osrd.api.project_signals.SignalProjectionEndpoint
import fr.sncf.osrd.api.standalone_sim.SimulationEndpoint
import fr.sncf.osrd.api.stdcm.STDCMEndpoint
```

Reply:
Nah just skipped this for some reason, good catch (code shouldn't work btw rn this is kinda weird)

Reply:
Fixed in https://github.com/OpenRailAssociation/osrd/pull/8318/commits/5cc66adeb8306365f0447fc9779f970ab69fc023.